src/concurrency.go

```diff
@@ -0,0 +1,52 @@
+package main
+
+import (
+	"fmt"
+	"net/http"
+)
+
+type response struct {
+	Code int
+	URL  string
+	Err  error
+}
+
+func main() {
```

Comment:
Add description here about program. For e.g. 
- Purpose of program
- How concurrency achieve here?
- Call out packages & special methods used in the program